pywinauto/unittests/test_HwndWrapper.py

```diff
@@ -319,6 +319,25 @@ def testGetProperties(self):
 #    def testCaptureAsImage(self):
 #        pass
 
```

Comment:
Disabling new tests since they would not work on AppVeyor environment. "Black screenshots" problem.